linker/linker_test.go

```diff
@@ -2511,29 +2525,28 @@ func TestLinkerValidation(t *testing.T) {
 				`,
 			},
 		},
-		"failure_extension_declaration_without_verification": {
+		"success_extension_declaration_without_verification": {
```

Comment:
Ops. I misread the `protoc` code. And I thought this test case confirmed I understood it okay: that the extension range must explicitly set `verification=DECLARATION` in order to add `declaration` values.

However, I had a typo in the field number. Fixing that, now `protoc` _succeeds_ for this case, so I had to update protocompile to follow suit.